src/constants/triggers.ts

```diff
@@ -32,99 +71,110 @@ export const taskRegexSelectors: RegexSelector[] = [
   },
 ];
 
+const failureTypeSubscriberOptions: string[] = [
+  "Any", // default
+  "Test",
+  "System",
+  "Setup",
+];
+
 export const failureTypeSubscriberConfig = {
   text: "Failure Type",
   key: "failure-type",
   type: "select",
-  options: {
-    any: "Any",
-    test: "Test",
-    system: "System",
-    setup: "Setup",
-  },
-  default: "any",
+  options: failureTypeSubscriberOptions,
 };
 
+const requesterSubscriberOptions: string[] = [
+  "Commit", // default
+  "Patch",
+  "Pull Request",
+  "Commit Queue",
+  "Periodic Build",
+];
+
 export const requesterSubscriberConfig = {
   text: "Build Initiator",
   key: "requester",
   type: "select",
-  options: {
-    gitter_request: "Commit",
-    patch_request: "Patch",
-    github_pull_request: "Pull Request",
-    merge_test: "Commit Queue",
-    ad_hoc: "Periodic Build",
-  },
-  default: "gitter_request",
+  options: requesterSubscriberOptions,
 };
 
 export const clearExtraFieldsInputCb = (accum: StringMap, eF: ExtraField) => ({
   ...accum,
   [eF.key]: "10",
 });
 
-export const projectTriggers: Trigger[] = [
+export const triggers: Trigger[] = [
   {
     trigger: TriggerType.OUTCOME,
     resourceType: ResourceType.VERSION,
     label: "Any Version Finishes",
     extraFields: [requesterSubscriberConfig],
+    index: 0,
```

Comment:
This approach seems somewhat error-prone as any updates to notification types could require updating all of these index fields and it would be easy to make a mistake. It also may be related to the bug Annie pointed out? Suggestion below...

Reply:
I already have work around that in the next for the next PR. I have a bunch of work adding onto this.

Reply:
The bug Annie pointed out is fixed now.